templates/definition/vehicle/volvo-connected.yaml

```diff
@@ -6,24 +6,28 @@ requirements:
     de: |
       Für die Nutzung mit EVCC benötigst du einen Volvo Account und einen Volvo Connected Car API Key. 
       Erstelle dazu auf der [Account Seite](https://developer.volvocars.com/account/) eine neue Applikation und speichere den primären VCC API Key ab. 
-      Veröffentliche nun deine Applikation und wähle unter "Scopes" die Berechtigungen "Connected Vehicle API -> conve:vehicle_relation" und "Energy API -> (alles)" aus.
-      Als Redirect URL kannst du erstmal "http://localhost:9999" (oder etwas anderes nicht erreichbares) verwenden. Der Authorisierungscode wird später händisch aus dem Browser gelesen und eingegeben.
+      Veröffentliche nun deine Applikation und wähle unter "Scopes" die Berechtigungen "Connected Vehicle API -> conve:vehicle-relation" und "Energy API -> (alles)" aus.
```

Comment:
Können wir die notwendigen minimalen Scopes noch ergänzen? Bei mehr Scopes läuft man bei Volvo in den Freigabeprozess...

Reply:
Es ist sogar gar nicht schlimm, wenn man mehr Scopes auswählt. Die müssen dann zwar freigegeben werden, aber die Scopes, die keine Freigabe benötigen, sind trotzdem nutzbar.

Reply:
Ist das so? Mein Testclient (das ist allerdings schon einige Zeit her) war nie funktionsfähig.

Reply:
Steht explizit so in der Beschreibung der App, nachdem man sie angelegt hat.

Reply:
Da hat sich wohl seit meinem letzten Versuch einiges geändert.